test/extension.test.ts

```diff
@@ -18,8 +18,8 @@ import * as path from 'path';
 const checkCompletionItemsForSpecificPosition = (
     cursorPositionForComponent: vscode.Position,
     proposals: Array<vscode.CompletionItem>,
-    itemsShouldExist: boolean = true,
-    fileToOpenInWorkspace: string = 'index.jsx'
+    fileToOpenInWorkspace: string = 'index.jsx',
+    itemsShouldExist: boolean = true
```

Comment:
Name of a bool variable should start with "is, can, has"

Reply:
I'll do this in a separate commit because I didn't add this in this PR